### DISCUSSION TITLE
[Feature] Separate description from the rest of the metadata in shared albums

### DISCUSSION THREAD

**Discussion by starlingfire** [immich-app/immich#13252] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The goal of this feature is to allow an album's shared link to control the displaying of the photo's description separately from displaying the rest of the metadata, so that photos' descriptions can be seen, but not the rest of the metadata. 

A general approach to this would be:
   - Add a toggle for "Show description" on the share create/edit link screen
   - Modify the shared page to:
      - Show the photo's description if the "Show description" toggle is enabled
      - Show the remainder of the metadata if the "Show metadata" toggle is enabled

Example of the "Edit link" page with the new toggle:
![image](https://github.com/user-attachments/assets/12665d92-958e-413c-889c-435a9f4fc45e)

Example of the info pane of an an image in a shared album:
![image](https://github.com/user-attachments/assets/e7a9353b-b6e2-4326-a3b6-9158928c7eff)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by starlingfire** [comment]
After playing around with a local instance and reviewing more issues, I think that a better approach for this would be to
1. Allow shared links to use slideshow mode
2. Display the description in slideshow mode, configurable by the viewer

I see that [a comment was made](https://github.com/immich-app/immich/issues/8383#issuecomment-2052677940) on #8383 confirming that slideshow option is not currently available on shared links, so perhaps this discussion could stand in for that, plus adding the description to the slideshow.

**Reply by timonrieger** [reply]
regarding 2, #8710
